docs/about/overview/radio-settings.mdx

```diff
@@ -25,13 +25,16 @@ Power limits will generally be lifted in the software if `is_licensed` is set to
 
 ## Europe Frequency Bands
 
+EU countries will generally be covered by the Radio Equipment Directive ([2014/53/EU](http://data.europa.eu/eli/dir/2014/53/2023-10-01))) and subsequent amendments, which is then enacted in local laws. Useful information on this is located [here](https://single-market-economy.ec.europa.eu/single-market/european-standards/harmonised-standards/radio-equipment_en).
+The bands below were selected based on the maximum power and the duty cycle availalbe. It may be possible to use other frequency bands for specific applications, but these are not currently supported "out of the box".
```

Comment:
```suggestion
The bands below were selected based on the maximum power and the duty cycle available. It may be possible to use other frequency bands for specific applications, but these are not currently supported "out of the box".
```